receiver/mysqlreceiver/metadata.yaml

```diff
@@ -320,6 +336,37 @@ metrics:
       input_type: string
       monotonic: true
       aggregation: cumulative
+  mysql.connection.errors:
+    enabled: true
+    description: Errors that occur during the client connection process.
+    unit: 1
+    sum:
+      value_type: int
+      input_type: string
+      monotonic: true
+      aggregation: cumulative
+    attributes: [connection_error]
+  mysql.mysqlx_connections:
+    enabled: true
```

Comment:
We need to start making new metrics as disabled by default. The exception is if they are considered particularly valuable, then we have to use a featuregate (if the component is beta or stable).

Reply:
Disabled all new metrics by default. We can enable them as defaults in separate PRs